CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Unreleased
 
 ## Expressions
-- Added support for boolean `==` and `!=` expressions, which can be used for things like negating expressions.
+- Added support for boolean `==` and `!=` expressions, which can be used for things like negating boolean expressions.
```

Comment:
Can we get a simple test for these as well?  I fear regression on expressions a lot.

Reply:
@mortoray: This is just a clarification of the changelog-message. There's already tests for these, as the feature landed earlier.

Reply:
yep, that was part of another change; it showed up here as I added the word `boolean` for clarification after seeing that and the new entry side-by-side.